### PR TITLE
Create Docker container for Kafka with notification installed

### PR DESCRIPTION
**Pull request by chc5**
Move notifications/ directory inside kafka/. Move kafka/ outside of lib/

### CODE REVIEW

**Comment by chc5** [comment]
I have not tested this container with the OpenCV container yet. I plan on doing so in the upcoming pull request

File: src/kafka/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# Install notifications and runs the script in the background
+COPY notifications notifications
+RUN cd notifications && npm install --production
+RUN cd notifications && npm start &
+
+# Copy the start script over
+COPY start start
+
+# 9042: Kafka port
```
Comment (melvinheks) [inline src/kafka/Dockerfile:37]: This comment says 9042?

File: src/kafka/notifications/src/index.ts [hunk 2]
```diff
@@ -2,7 +2,8 @@ import * as fs from "fs";
 import sendEmail from './sendEmail';
```
Comment (melvinheks) [inline src/kafka/notifications/src/index.ts:2]: This file uses 4 spaces while sendmail uses 2 spaces for indentation

Comment (chc5) [reply src/kafka/notifications/src/index.ts]: This file has a lot of inner function args that make it seem like it's a 4 space indentation.

File: src/kafka/Dockerfile (hunk 1 above)
Comment (chc5) [reply src/kafka/Dockerfile]: fixed

**Pull request merged**